index.js

```diff
@@ -42,8 +42,20 @@ function fileToDataURI(filePath) {
     filePath = filePath && filePath.getValue() || filePath;
     return loadFile(filePath).then((file) => {
         var mimeType = mime.lookup(file.normalizedPath);
-        var base64 = file.content.toString('base64');
-        return `data:${mimeType};base64,${base64}`;
+
+        var encoded = 'base64,' + file.content.toString('base64');
+
+        // Try URL encoding to see if it saves bytes.
+        let utf8_decoded = file.content.toString('utf8');
```

Comment:
Could you please use camel case as the rest of the code?

Reply:
Whoops, fixed!